### PR TITLE
Add missing getProcessorToken that should have been included in the l…

### PR DESCRIPTION
**Pull request by paulpearcy**
…ast commit

### CODE REVIEW

**Comment by paulpearcy** [comment]
The deployed tool wasn't working for downloading assest/sending statuses. I have no idea how this change was missed in the initial PR, but obviously an oversight on my part.

File: server/src/app/lti-token-service.js [hunk 1]
```diff
@@ -48,5 +48,32 @@ export const getCachedLTIToken = async (nonce, clientId, scope) => {
...
+
+  const tokenUrl = getAppById(clientId).setup.jwtUrl;
+  let token =  await getLTIToken(clientId, tokenUrl, scope, null);
+  tokenCache[clientId][scope] = {
+    'expiration': now + 1000 * 60 * 30,
+    'token': token
+  }
+
+  return token;
+}
```
Comment (varju) [inline server/src/app/lti-token-service.js:79]: Mostly cosmetic:
```suggestion
  };

  return token;
};
```

File: server/src/app/lti-token-service.js [hunk 2]
```diff
@@ -48,5 +48,32 @@ export const getCachedLTIToken = async (nonce, clientId, scope) => {
 };
 
 const cacheToken = async (token, nonce) => {
-  await insertNewAuthToken(nonce, `${token}`, 'lti_token');
+  if (nonce != null) {
```
Comment (varju) [inline server/src/app/lti-token-service.js:51]: Would this be better just checking whether nonce is truthy? I don't see any other explicit equality checks for null in this code.
```suggestion
  if (nonce) {
```

File: server/src/app/lti-token-service.js [hunk 3]
```diff
@@ -48,5 +48,32 @@ export const getCachedLTIToken = async (nonce, clientId, scope) => {
...
+    await insertNewAuthToken(nonce, `${token}`, 'lti_token');
+  }
 };
+
+// Keep a simple in memory cache of processor tokens
+let tokenCache = {};
```
Comment (varju) [inline server/src/app/lti-token-service.js:57]: Mostly cosmetic:
```suggestion
const tokenCache = {};
```

**Review by varju — APPROVED** [review]

**Pull request merged**